src/components/Nav/UserDropdown.tsx

```diff
@@ -51,11 +54,13 @@ class UserDropdown extends React.Component<UserProps, UserState> {
   }
 
   timeLeft = (): number => {
-    const nowDate = new Date().getTime();
-    if (this.props.sessionTimeOut - nowDate < 1) {
+    const expiresOn = moment(this.props.session.expiresOn).toDate();
```

Comment:
If we are going to use `moment` there is no need for `new Date()` just use `moment()` and its comparisons and arithmetic it's much more concise. There should be no need for `moment(xxxx).toDate()` anywhere with moment conversions.